lib/cisco_node_utils/cmd_ref/vrf.yaml

```diff
@@ -41,6 +41,25 @@ destroy:
   ios_xr:
     set_value: 'no vrf <vrf>'
 
+mhost_ipv4:
```

Comment:
Did you consider combining mhost_ipv4 and mhost_ipv6?

See https://cdtg-kickstart.cisco.com/sunstone-docs/yaml/vrfsub.yaml for sample yaml strings.